hardware: info for not supported platforms (#230)
seL4/docs: Hardware/index.md

More information can be found on the [Verified Configurations](../projects/sel4/verified-configurations.md) page.

### Not in the lists below?

If the platform, architecture, feature that you are after is not listed below, your
options are:

* contact one of the seL4 Foundation [Endorsed Services
  Providers](https://sel4.systems/Foundation/Services/) to get commercial
  support to develop such a port or feature;
* check the [roadmap](../projects/roadmap.html) for any planned contributions,
  from the seL4 Foundation or larger community, such on any new architecture
  ports, new large formal verifications, or large or fundamental new features.
* use one of our [communication channels](https://sel4.systems/contact/) to
  inquire information, from the seL4 Foundation or the larger community, on

>   implementing such a port or feature;

Author: @lsf37 thanks I have reworded it. Please let me know if it's better

>   implementing such a port or feature;

Reviewer: have tweaked it a bit more (see below)

* develop the seL4 port or feature yourself, following our [guidelines for

>   contributing kernel code](../projects/sel4/kernel-contribution.html).

Reviewer: Same as above, we are not an OSS project that will just take new features (esp without verification) -- it needs discussion and an RFC first, and a lot of the time it turns out that existing features are sufficient to solve the problem. The linked page says this (RFC/discussion), but the list here makes it sound like we do want to encourage contributions without talking first.

Platform/BSP ports are different, because they are easy to maintain, but they still need tests and CI, so even there it is not completely unproblematic. New architectures are unlikely to get accepted without major maintenance and expertise commitment. Features are possible, but need discussion and design.

Author: @lsf37 thanks I have added explanation in the intro, re-ordered and rephrased a bit the list. Please let me know if it's better

More information can be found on the [Verified Configurations](../projects/sel4/verified-configurations.md) page.

### Not in the lists below?

If the platform, architecture, feature that you are after is not listed below,

> you have several options, listed below. It is important to note however that, as
> explained in the guidelines linked below, new ports or features will require

Reviewer: ```suggestion
you have several options, listed below. It is important to note however that, as
explained in the guidelines linked below, contributing new ports or features will require
```

compelling arguments, discussion with the technical community (including through
Request-For-Comments), as well as testing requirements and maintenance/expertise
commitment, to a degree depending on the nature of the contribution.

* You can check the [roadmap](../projects/roadmap.html) for any planned
  contributions, from the seL4 Foundation or larger community, such on any new
  architecture ports, new large formal verifications, or large or fundamental
  new features;

> * You can inquire the seL4 community, through one of our [communication
>   channels](https://sel4.systems/contact/) as to whether someone is developing
>   such a port or feature already, or as to the general interest in discussing

Reviewer: "inquire" doesn't quite work grammatically. Maybe the following?
```suggestion
* You can contact the seL4 community through one of our [communication
  channels](https://sel4.systems/contact/) to ask whether someone is developing
  such a port or feature already, or if there is general interest in discussing
```

you have several options, listed below. It is important to note however that, as
explained in the guidelines linked below, new ports or features will require
compelling arguments, discussion with the technical community (including through
Request-For-Comments), as well as testing requirements and maintenance/expertise
commitment, to a degree depending on the nature of the contribution.

* You can check the [roadmap](../projects/roadmap.html) for any planned
  contributions, from the seL4 Foundation or larger community, such on any new
  architecture ports, new large formal verifications, or large or fundamental
  new features;

* You can inquire the seL4 community, through one of our [communication
  channels](https://sel4.systems/contact/) as to whether someone is developing
  such a port or feature already, or as to the general interest in discussing
  such such a new port or feature;

* If you are in a position to develop the seL4 port or feature yourself, you
  should follow our [guidelines for contributing kernel
  code](../projects/sel4/kernel-contribution.html), which details the
  implications and expectations;

> * You can also contact one of the seL4 Foundation [Endorsed Services

Reviewer: I liked that the service providers were high up in the list before. Unless there is a good reason not to, we should keep them at second or first place.

Author: back in 1st position
